packages/material-ui/src/Portal/Portal.js

```diff
@@ -17,65 +17,60 @@ function getOwnerDocument(element) {
  * Portals provide a first-class way to render children into a DOM node
  * that exists outside the DOM hierarchy of the parent component.
  */
-class Portal extends React.Component {
-  componentDidMount() {
-    this.setMountNode(this.props.container);
+function Portal(props) {
+  const {children, disablePortal, onRendered, container} = props;
+  const [, forceUpdate] = React.useState(0);
```

Comment:
I might be pain in the ass.... Is this a good practice to desctructure array in such way? I would rather prefer ```React.useState(0)[1]```

Reply:
https://twitter.com/dan_abramov/status/1118301474876948480
😄